src/Analyzers/VerifyShouldBeUsedOnlyForOverridableMembersAnalyzer.cs

```diff
@@ -0,0 +1,195 @@
+                return propertySymbol.IsOverridable() || propertySymbol.IsTaskOrValueResultProperty(knownSymbols);
+
+            case IMethodSymbol methodSymbol:
+                return methodSymbol.IsOverridable();
+
+            default:
+                // If it's not a property or method, it can't be mocked. This includes fields and events.
+                return false;
+        }
+    }
+
+    /// <summary>
+    /// Attempts to resolve the symbol representing the member (property or method)
+    /// being referenced in the Verify(...) or VerifySet(...) call. Returns null if it cannot be determined.
+    /// </summary>
+    private static ISymbol? TryGetMockedMemberSymbol(IInvocationOperation moqVerifyInvocation, MoqKnownSymbols knownSymbols)
+    {
+#pragma warning disable S125 // Sections of code should not be commented out
+        // Usually the first argument to a Moq Verify(...) is a lambda expression like x => x.Property
+        // or x => x.Method(...). For VerifySet, it's an Action<T> lambda: x => { x.Property = ...; }
+#pragma warning restore S125 // Sections of code should not be commented out
+        if (moqVerifyInvocation.Arguments.Length == 0)
+        {
+            return null;
+        }
+
+        // For VerifySet, the lambda may be in argument 1 (second argument)
+        int lambdaArgIndex = 0;
+        if (moqVerifyInvocation.TargetMethod.IsInstanceOf(knownSymbols.Mock1VerifySet) && moqVerifyInvocation.Arguments.Length > 1)
+        {
+            lambdaArgIndex = 1;
+        }
+
+        IOperation argumentOperation = moqVerifyInvocation.Arguments[lambdaArgIndex].Value;
+        argumentOperation = argumentOperation.WalkDownImplicitConversion();
+
+        // Handle delegate conversions (e.g., VerifySet(x => { ... }))
+        if (argumentOperation is IDelegateCreationOperation delegateCreation &&
+            delegateCreation.Target is IAnonymousFunctionOperation lambdaOp)
+        {
+            argumentOperation = lambdaOp;
+        }
+
+        if (argumentOperation is IAnonymousFunctionOperation lambdaOperation)
+        {
+            // For VerifySet, the lambda body is a block with an assignment statement.
+            if (moqVerifyInvocation.TargetMethod.IsInstanceOf(knownSymbols.Mock1VerifySet))
+            {
+                ImmutableArray<IOperation> bodyOps = lambdaOperation.Body.Operations;
+                foreach (IOperation op in bodyOps)
+                {
+                    if (op is IExpressionStatementOperation exprStmt)
+                    {
+                        IAssignmentOperation? assignOp = exprStmt.Operation as IAssignmentOperation
+                            ?? exprStmt.Operation as ISimpleAssignmentOperation;
+
+                        if (assignOp?.Target is IPropertyReferenceOperation propRef)
+                        {
+                            return propRef.Property;
+                        }
+                    }
+                }
+
+                return null;
```

Comment:
This line is also uncovered. Write a unit test for this if the scenario is valid; otherwise, refactor.